email-filters.html

```diff
@@ -0,0 +1,166 @@
+    <p>Here are just a few of the things you can do with Sieve:</p>
+    <hr>
+    <ul>
+        <li>
+            Filter out spam and unwanted emails. Sieve filters can be used to filter out spam and unwanted emails based on their sender, subject, or content.
+        </li>
+        <hr>
+        <li>
+            Organize your inbox automatically. Sieve filters can be used to automatically move or copy your emails into different folders based on their sender, subject, or content.
+        </li>
+        <hr>
+        <li>
+            Forward important emails to specific people. Sieve filters can be used to automatically forward important emails to specific people, such as your manager or your assistant.
+        </li>
+        <hr>
+        <li>
+            Sieve filters can be used to create custom notifications and alerts for important emails, such as those from your boss or clients. Unlike simply forwarding the email, these notifications and alerts can provide immediate and specific information, ensuring you stay updated without needing to sift through your inbox.
+        </li>
+        <hr>
+    </ul>
+    <br/>
+    <p> To get started with Sieve in Cypht, simply go to your config file (hm3.ini for Cypht 1.4.x or .env for Cypht 2.0.x) and enable the Sieve filter engine by enabling modules[ ]=sievefilters or by adding sievefilters on CYPHT_MODULES variable like:
+
+        <pre>modules[]=sievefilters #For Cypht 1.4.x
+CYPHT_MODULES="sievefilters" #For Cypht 2.0.x</pre>
+        Once you have enabled the Sieve filter engine, you can create and manage your filters in the Filters section of the Cypht interface. You can create new filters, edit existing filters, and delete filters as needed.
+    </p>
+    <h3>How to create a Sieve filter in Cypht?</h3>
+    <p>
+        To create a Sieve filter in Cypht follow these steps:
+    </p>
+
+    <ul>
+        <li>
+            Go to <b>Settings</b> tab in the sidebar.<br/>Then click on <b>Filters</b> tab, select the E-mail account you want for filters and click on the "Add Filter" button.
+        </li>
+        <li>
+            In the "Filter name" field,<br>
+            enter a name for your new filter. Then provide other details of the filter: priority(It defines an order value (or priority value) for your filter). Note that Filters will run from lowest to highest priority value. and test (which will help to combine two or more conditions.)
+        </li>
+        <li>
+            In the "Conditions section", <br>
+            you can add conditions to your filter. You can add multiple conditions to your filter. You can add conditions based on the sender, subject, body, etc. Check out the <a href="documentation.html">documentation</a> page for other information.
```

Comment:
you can add conditions to your filter. You can add multiple conditions to your filter. -> you can add one many conditions to your filter.